ui/src/app/views/workspace-edit/component.tsx

```diff
@@ -449,78 +532,189 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
-                  value={this.state.workspace.researchPurpose.population}
-                  onChange={v => this.updateWorkspaceCategory('population', v)}/>
-              <WorkspaceCategory item={ResearchPurposeItems.commercialPurpose}
-                  value={this.state.workspace.researchPurpose.commercialPurpose}
-                  onChange={v => this.updateWorkspaceCategory('commercialPurpose', v)}/>
+        </WorkspaceEditSection>
+        <WorkspaceEditSection
+          header={researchPurposeQuestions[1].header}
+          text={researchPurposeQuestions[1].description} required>
+          <TextArea value={this.state.workspace.researchPurpose.softwareChoice}
+                    onChange={v => this.updateResearchPurpose('softwareChoice', v)}
+                    style={{marginTop: '0.5rem'}}/>
+        </WorkspaceEditSection>
+        <WorkspaceEditSection
+          header={researchPurposeQuestions[2].header}
+          text={researchPurposeQuestions[2].description} required>
+          <TextArea value={this.state.workspace.researchPurpose.intendedStudy}
+                    onChange={v => this.updateResearchPurpose('intendedStudy', v)}
+                    style={{marginTop: '0.5rem'}}/>
+        </WorkspaceEditSection>
+        <WorkspaceEditSection header={researchPurposeQuestions[3].header}
+                              text={researchPurposeQuestions[3].description} required>
+          <TextArea value={this.state.workspace.researchPurpose.anticipatedFindings}
+                    onChange={v => this.updateResearchPurpose('anticipatedFindings', v)}
+                    style={{marginTop: '0.5rem'}}/>
+        </WorkspaceEditSection>
+        <WorkspaceEditSection required header={researchPurposeQuestions[4].header}>
+          <Link onClick={() => this.setState({showUnderservedPopulationDetails:
+              !this.state.showUnderservedPopulationDetails})} style={{marginTop: '0.5rem'}}>
+            More info on underserved populations
+            {this.state.showUnderservedPopulationDetails ? <ClrIcon shape='caret' dir='up'/> :
+              <ClrIcon shape='caret' dir='down'/>}
+          </Link>
+          {this.state.showUnderservedPopulationDetails && <div style={styles.text}>
+            A primary mission of the AoU Research Program is to include research participants
+            who are medically underserved or are historically underrepresented in Biomedical
+            Research, or who, because of systematic social disadvantage, experience health
+            disparities.  As a way to assess the research being conducted with a focus on these
+            populations, AoU requires that you indicate the demographic categories you intend
+            to focus your analysis on.
+          </div>}
+          <div style={{marginTop: '0.5rem'}}>
+            <RadioButton name='population' style={{marginRight: '0.5rem'}}
+              onChange={v => this.updateResearchPurpose('population', true)}
```

Comment:
The spacing on this radio button vs the lower one is different. Should probably be consistent (surprised our linter didn't get upset)